frontend/src/shared/components/SelectorCard.js

```diff
@@ -122,8 +111,13 @@ class SelectorCard extends Component {
               md="1"
               block
               onClick={this.toggleExpand}
+              style={{ alignItems: 'center' }}
             >
-              {!expanded ? 'A' : 'V'}
+              {!expanded ? (
+                  <FontAwesomeIcon icon={FaCaretRight} size='lg'/>
+                ) : (
+                  <FontAwesomeIcon icon={FaCaretLeft} className='mr-2' size='lg'/>
+              )}
```

Comment:
these should probably be right for closed and down for open, not left and right, sorry

Reply:
Changed